src/dstack/_internal/server/services/logs/aws.py

```diff
@@ -271,7 +269,7 @@ def _runner_log_event_to_cloudwatch_event(
     ) -> _CloudWatchLogEvent:
         return {
             "timestamp": runner_log_event.timestamp,
-            "message": b64encode_raw_message(runner_log_event.message),
+            "message": runner_log_event.message.decode(),
```

Comment:
.decode() will raise an exception if bytes are invalid utf-8. If it's handled by the runner, please leave a comment since it's very non-obvious when the runner itself returns base64 encoded bytes instead of strings.